### ISSUE TITLE
Compatibility with Joomla 4?

### ISSUE THREAD

**Issue by machadoug** [elkuku/EasyCreator#59]
Is this extension going to be compatible with Joomla 4?

**Comment by elkuku** [comment]
Yes I hope so.
I am not sure when, or even how, but any help would be greatly appreciated 😉

**Comment by MrBeet** [comment]
I hope so much will be running with J4. Like to say many thanks for your great Job done in last years. I like this tools so much....without i never created a Joomla Component or Module....so thanks thanks thanks!!! Kind regards  beat

**Comment by ssnobben** [comment]
yes Joomla 4 beta now released and hope to see this support for Joomla 4 too. :) all the best!
